src/features/customerView/CustomerViewPageContainer.tsx

```diff
@@ -4,14 +4,14 @@ import { useTranslation } from 'react-i18next';
 import { useQuery } from '@apollo/react-hooks';
```

Comment:
The component name in this file is wrong (`IndividualHarborPageContainer`)

Reply:
Fixed